docs/stories/Inputs/NumberInput.stories.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from 'react';
+
+import { useArgs } from '@storybook/preview-api';
+import { Meta, StoryObj } from '@storybook/react';
+import { NumberInput, Flex, Button, Field, FieldLabel, FieldHint, FieldError } from '@strapi/design-system';
+import { default as outdent } from 'outdent';
+
+const meta: Meta<typeof NumberInput> = {
+  title: 'Design System/Inputs/NumberInput',
+  component: NumberInput,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof NumberInput>;
+
+const Template: Story = {
+  render: ({ value, ...props }) => {
+    const [, updateArgs] = useArgs();
+
+    return (
+      <Flex direction="column" alignItems="stretch" gap={4}>
+        <NumberInput
+          {...props}
+          placeholder="This is a content placeholder"
```

Comment:
Let's try to be real with the stories, this realistically could say "Price (GBP)"